### DISCUSSION TITLE
Endpoint de dados climáticos

### DISCUSSION THREAD

**Discussion by Davidson-Souza** [BrasilAPI/BrasilAPI#233] [Ideas]
Fala, turma! Tudo certo?
Já que a proposta é ser uma aplicação que condensa vários endpoint úteis em uma coisa só, acredito que um endpoint para buscar previsão do tempo seja algo razoável de ser pensado.
Aqui o exemplo de uma API bacana para buscar por esse tipo de dado: [HGBrasil](https://hgbrasil.com/status/weather)

**Comment by Nikolassantos** [comment]
> Fala, turma! Tudo certo?
> Já que a proposta é ser uma aplicação que condensa vários endpoint úteis em uma coisa só, acredito que um endpoint para buscar previsão do tempo seja algo razoável de ser pensado.
> Aqui o exemplo de uma API bacana para buscar por esse tipo de dado: [HGBrasil](https://hgbrasil.com/status/weather)

Achei muito boa essa API mencionada , acho que seria valido emplementarmos.

**Comment by gabrielbahniuk** [comment]
@Davidson-Souza @Nikolassantos voces ja chegaram a implementar algo? Gostaria de participar também :-)

Eu vi que essa API tem a parte financeira também: https://console.hgbrasil.com/documentation/finance

Talvez seja válido analisar se ja da pra fazer tanto a parte climática quanto a parte da BOVESPA (esta última em outra issue)

**Comment by Nikolassantos** [comment]
@gabrielbahniuk , @Davidson-Souza  vou fazer a implementação e anexar o PR pra voces darem oma olhada

**Reply by RodriAndreotti** [reply]
Cara, já tem um PR meu buscando dados climáticos provenientes do CPTEC - INPE, Será que vale a pena trazer outro serviço? Ou talvez tentarmos ajustar como fallback?

PS.: Verificando a HG Brasil, notei que é um serviço pago, e bem restrito quanto aos limites de consulta diários:
https://hgbrasil.com/apis/planos

Será que vale o esforço para trazê-lo?

**Comment by LorhanSohaky** [comment]
@RodriAndreotti , acho que a sua implementação do CPTEC resolve isso, né? #181

**Reply by LorhanSohaky** [reply]
@Davidson-Souza , podemos marcar como resolvido?

**Reply by RodriAndreotti** [reply]
Acredito que sim, @LorhanSohaky!
Se faltar algo talvez possamos fazer num próximo PR.
